static/lang/en.json

```diff
@@ -1565,6 +1565,10 @@
                 "StowingLabel": "Is Stowing Container?"
             },
             "CannotAddType": "{type} items cannot be added to this actor.",
+            "Categories": {
+                "Physical": "Physical Items",
+                "Other": "Other Items"
+            },
```

Comment:
Move to `PF2E.Item.CreationDialog` or the like--top-level entries should be rare

Reply:
I saw there were a few there already, and categories weren't just creation dialog as a concept (if subtypes occur, categories are the super types). But sure.